iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/telemetry/SendMessagesTests.java

```diff
@@ -118,54 +118,6 @@ public void sendMessagesWithUnusualApplicationProperties() throws Exception
         this.testInstance.identity.getClient().closeNow();
     }
 
-    @Test
-    @ContinuousIntegrationTest
-    public void sendMessagesOverAmqpsMultithreaded() throws InterruptedException, IOException, IotHubException, GeneralSecurityException, URISyntaxException
```

Comment:
We already have a test that ensures d2c telemetry works over AMQP and AMQP_WS, so the only interesting attribute to this test was the multithreaded portion. As discussed elsewhere in this PR, multithreaded tests aren't reliable enough at testing thread safety to be worthwhile.